SwiftWisdomTests/Rx/CollectionVariableTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+import RxSwift
+@testable import SwiftWisdom
+
+class CollectionVariableTests: XCTestCase {
```

Comment:
I'm removing this. It is a whole library that should just be updated for swift3, it doesn't feel right to just copy and absorb it.